unicoremc/models.py

```diff
@@ -367,7 +432,8 @@ def create_pyramid_settings(self):
                 self.available_languages.all(),
                 self.default_language or Localisation._for('eng_GB'),
                 self.ga_profile_id,
-                self.hub_app()
+                self.hub_app(),
+                self.all_repos()[0].name()
```

Comment:
No. `all_repos()[0]` is the same as `own_repo()` if the project has its own repo. But it might not have its own repo.